babybuddy/settings/docker.py

```diff
@@ -4,13 +4,14 @@
 # Database
 # https://docs.djangoproject.com/en/3.0/ref/settings/#databases
 
+# Load settings from env file / variables with fallback defaults to support current psql deployment
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.postgresql_psycopg2',
-        'NAME': 'postgres',
-        'USER': 'postgres',
-        'PASSWORD': os.environ.get('POSTGRES_PASSWORD'),
-        'HOST': 'db',
-        'PORT': 5432,
+        'ENGINE': os.getenv('DB_ENGINE') or 'django.db.backends.postgresql_psycopg2',
+        'NAME': os.getenv('DB_NAME') or 'postgres',
+        'USER': os.getenv('DB_USER') or 'postgres',
+        'PASSWORD': os.environ.get('DB_PASSWORD'),
```

Comment:
Two final items

1. Lets fall back on `POSTGRES_PASSWORD` for now. This way we don't break any existing docker compose configurations unexpectedly.
2. For new deployments, lets also update [`docker-compose.example.yml`](https://github.com/babybuddy/babybuddy/blob/master/docker-compose.example.yml) to use the new environment variable in the `app` section.

Reply:
Newest set of commits reflect those two changes